packages/contracts-periphery/contracts/universal/op-nft/AttestationStation.sol

```diff
@@ -44,31 +44,57 @@ contract AttestationStation is Semver {
     );
 
     /**
-     * @custom:semver 1.0.0
+     * @custom:semver 1.1.0
      */
-    constructor() Semver(1, 0, 0) {}
+    constructor() Semver(1, 1, 0) {}
+
+    /**
+     * @notice Allows anyone to create an attestation.
+     *
+     * @param _about Address that the attestation is about.
+     * @param _key   A key used to namespace the attestation.
+     * @param _val   An arbitrary value stored as part of the attestation.
+     */
+    function attest(
```

Comment:
I did it like this so that `attest` could be `external` instead of `public`. It was cheaper to have `attest` accept `calldata` rather than `memory`. I don't think its  a huge readability improvement but if you are strongly opinionated then I'm ok with it, it'll just add more gas usage

Reply:
Why is calldata cheaper than memory?

Also IMO one could argue this is more readable and I don't think it's worth changing if there is a gas impact myself